Content.Server/GameObjects/Components/Items/Storage/StorageFillComponent.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using NFluidsynth;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.Interfaces.Serialization;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.Serialization;
+using Serilog;
+using Logger = Robust.Shared.Log.Logger;
+
+namespace Content.Server.GameObjects.Components.Items.Storage
+{
+    [RegisterComponent]
+    internal sealed class StorageFillComponent : Component, IMapInit
+    {
+        [Dependency] private readonly IEntityManager _entityManager;
+
+        public override string Name => "StorageFill";
+
+        private List<PrototypeItemData> _contents = new List<PrototypeItemData>();
```

Comment:
This initial setting probably isn't necessary as we will overwrite it anyway in ExposeData.
The only thing that would access that array before ExposeData would be Initialize (which we don't use here).